lib/zendesk_apps_support/location.rb

```diff
@@ -14,6 +14,7 @@ def initialize(attrs)
       Location.unique_ids.add @id
       @name = attrs.fetch(:name)
       @orderable = attrs.fetch(:orderable)
+      @collapsible = attrs.fetch(:collapsible)
```

Comment:
I wonder if we could have implemented collapsible similar to `v2_only`, defaulting to `false`, so it would only need to be declared for a Location where it was `true`?

Reply:
We can make it default to either value in the constructor, no?

Reply:
Yeah I did think about this. For consistency I might update this so it's set the same way as `v2_only`

Reply:
> We can make it default to either value in the constructor, no?

I described as defaulting to `false`, cos' it looks like (similar to `v2_only`), the majority case is `false`, meaning less lines of code to declare `true` where needed

Reply:
Sure, I was thinking the same could be applied to the `orderable` flag.